### DISCUSSION TITLE
MIssing Snowflake DB to create connection

### DISCUSSION THREAD

**Discussion by LmG7119** [dbeaver/dbeaver#34779] [Q&A] [answered]
### Description

I have created snowflake connections before and still have them but when I try to created new one I see no Snowflake option anymore.  Any ideas who to restore it?

![image](https://github.com/dbeaver/dbeaver/assets/8144582/c4fc874a-e367-48cf-b10d-53ea6a57498b)

### DBeaver Version

Community EdditionVersion 24.1.0.202406091028

### Operating System

Windows 10 Pro

**Comment by LonwoLonwo** [comment]
Hello @LmG7119 

It is strange.
Could you please change the category to the "All"?

![2024-07-11 11_42_34-MIssing Snowflake DB to create connection · Issue #34765 · dbeaver_dbeaver](https://github.com/dbeaver/dbeaver/assets/45152336/6f9e4f02-b6c3-4deb-9e55-837722a76345)

**Comment by E1izabeth** [comment]
I can't reproduce it 
![image](https://github.com/dbeaver/dbeaver/assets/28875055/a47a135f-3826-41bc-8226-80d3fc633a6d)

Could you check if you see it in driver manager?
![image](https://github.com/dbeaver/dbeaver/assets/28875055/b833441c-c904-4b48-a2d6-814aa8e151ca)

![image](https://github.com/dbeaver/dbeaver/assets/28875055/f27a60ac-c51e-4e33-9b88-36c14c50f28e)

**Comment by ShadelessFox** [comment, marked as answer]
Hello @LmG7119,

Are you still able to connect to your existing connections? 

Additionally, please take a look at the driver manager. Maybe the driver was deleted somehow:

![Image](https://github.com/dbeaver/dbeaver/assets/35821147/19c5fe1f-097d-468d-8c1a-b0084f6a54e2)

**Comment by LmG7119** [comment]
The un-delete did the trick.
